amazonredshift_blueprints/upload_file.py

```diff
@@ -119,25 +125,96 @@ def combine_folder_and_file_name(folder_name, file_name):
 
     return combined_name
 
+def map_datatypes(pandas_type:str) -> str:
+    """ Maps the pandas datatype to the corresponding SQL data type
+    Args:
+        pandas_type (str): the pandas datatype
+    """
+    if pandas_type == 'object':
+        return 'VARCHAR(255)'
+    elif pandas_type == 'int64':
+        return 'INT'
+    elif pandas_type == 'float64':
+        return 'FLOAT'
+    elif pandas_type == 'bool':
+        return 'BOOLEAN'
+    elif pandas_type == 'datetime64':
+        return 'DATETIME'
+    elif pandas_type == 'timedelta[ns]':
+        return 'INTERVAL'
+    elif pandas_type == 'category':
+        return 'VARCHAR(255)'
+    else:
+        return 'VARCHAR(255)'
+
+def create_table_query(table_name:str, df:pd.DataFrame, database:str, schema:str = 'public') -> str:
```

Comment:
I'm not 100% sure we can default to public, but I'm not going to dissaprove this.

Some databases use the user's default schema if no schema is provided. Not sure if that applies to Redshift.